docker-compose.yml

```diff
@@ -6,7 +6,7 @@ networks:
 
 services:
   influxdb:
-    image: influxdb:latest
```

Comment:
What is the reason to change the version of influxdb?

Reply:
@sureshk2a Because influxdb v2 (the `latest` version) isn't supported yet without modifications. You'll get the following error:

`Couldn't write stats, InfluxDB v2.x isn't supported by this output, if you are using it you may consider to use the extension https://github.com/grafana/xk6-output-influxdb  error="{\"code\":\"unauthorized\",\"message\":\"unauthorized access\"}" output=InfluxDBv1`